charts/jenkins/CHANGELOG.md

```diff
@@ -12,11 +12,39 @@ Use the following links to reference issues, PRs, and commits prior to v2.6.0.
 The changelog until v1.5.7 was auto-generated based on git commits.
 Those entries include a reference to the git commit to be able to get more details.
 
+## 5.0.0
```

Comment:
Need to include the breaking changes to adminSecrets at the time of writing this comment.